src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -46,19 +34,18 @@ def serialize(self, obj, attrs, user):
             "name": obj.name,
             "organizationId": six.text_type(obj.organization_id),
             "status": obj.status,
-            # TODO: Remove when frontend isn't using
-            "thresholdType": 0,
-            "dataset": obj.dataset,
-            "query": obj.query,
+            "dataset": obj.snuba_query.dataset,
+            "query": obj.snuba_query.query,
+            "aggregate": obj.snuba_query.aggregate,
+            # These fields are deprecated. Once we've moved over to using aggregate
+            # entirely we can remove
             "aggregation": obj.aggregation,
             "aggregations": [obj.aggregation],
-            "timeWindow": obj.time_window,
-            "environment": attrs.get("environment", []),
-            "resolution": obj.resolution,
-            # TODO: Remove when frontend isn't using
-            "alertThreshold": 0,
-            # TODO: Remove when frontend isn't using
-            "resolveThreshold": 0,
+            # TODO: Start having the frontend expect seconds
+            "timeWindow": obj.snuba_query.time_window / 60,
```

Comment:
Isn't this already in seconds? Looks like we're storing it as seconds.

Reply:
Yeah, but due to a bad decision I made early on we store the `time_window` on the alert rule in minutes, and the frontend passes minutes. I'm gradually shifting all of the backend to use seconds, and will then work on having the frontend change too.